action.yml

```
@@ -261,6 +264,16 @@ runs:
           && '--retry-on-error --continue-on-error --diff'
           || ''
       }}
+      ${{
+          inputs.testing-type == 'integration' && inputs.tags
+          && format('--tags {0}', inputs.tags)
```

You'll probably want to account for values with spaces:
```suggestion
          && format('--tags "{0}"', inputs.tags)
```